common/tools/dev-tool/src/commands/customization/apply-v2.ts

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license
+
+import path from "path";
+
+import { resolveProject } from "../../util/resolveProject";
+import { createPrinter } from "../../util/printer";
+import { leafCommand } from "../../framework/command";
+import { makeCommandInfo } from "../../framework/command";
+
+import { execSync } from "child_process";
+import fs from "fs-extra";
```

Comment:
do we need `fs-extra`? I think everything you're doing is supported by `fs/promises`, e.g. https://nodejs.org/docs/latest/api/fs.html#fspromisesmkdtempprefix-options